packages/slider/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@reach/slider",
+  "version": "0.3.0-beta.0",
+  "description": "Accessible React Slider Component",
+  "main": "index.js",
+  "module": "es/index.js",
+  "scripts": {
+    "test": "echo \"Write some tests you bum!\"",
+    "build": "node ../../shared/build-package",
+    "lint": "eslint . --max-warnings=0"
+  },
+  "dependencies": {
+    "@reach/auto-id": "0.3.0-alpha.0",
+    "@reach/utils": "0.3.0-alpha.0",
+    "warning": "^4.0.3"
+  },
+  "peerDependencies": {
+    "prop-types": "^15.7.2",
+    "react": "^16.9.0",
+    "react-dom": "^16.9.0"
+  },
+  "author": "Ryan Florence <@ryanflorence>",
+  "contributors": [
+    "Chance Strickland <@chancethedev>",
+    "Michael Jackson <@mjackson>"
+  ],
```

Comment:
Let's just use `"author": "React Training <hello@reacttraining.com>"` instead of individual `author` and `contributors` fields going forward. We're like a band. We *all* write the songs!

Reply:
Updated